docs/source/api/deliveryservices_required_capabilities.rst

```diff
@@ -13,11 +13,11 @@
 .. limitations under the License.
 ..
 
-.. _to-api-deliveryservice-server-capabilities:
+.. _to-api-deliveryservice-required-capabilities:
```

Comment:
should this be deliveryservices-required-capabilites? ( missing s on end of deliveryservice)

Reply:
I guess so. What does this line get used for in the docs anyway?

Reply:
I believe it is used to reference from other pages

Reply:
yes, that's a reference label. You can, in practice put them anywhere to act as labels ("anchors" in HTML) but truly "label-able" things let you do
```rst
.. _section:

*************
SECTION TITLE
*************
Some paragraph text

.. _some-thing:

lorem ipsum dolor etc.

.. _figure:
.. figure:: jay-peg.png
    :caption: Figure caption

.. _table:
.. table:: My table

    A | B
    -+-
    1| 0

:ref:`section` (renders as "SECTION TITLE")
:ref:`figure` (renders as "Figure caption")
:ref:`table` (renders as "My table")
:ref:`some-thing` (this is an error - no title)
:ref:`Some thing <some-thing>` (Renders as a link to some-thing with the text "Some thing")
```